src/database/migrations/20240122182032-create-auth-table.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Auths', {
```

Comment:
the casing of table names are inconsistent (see settings table's name).

let's keep all of them in snake_case